Header header
geometry_msgs/Vector3 pos_est
geometry_msgs/Vector3 vel_est
int32 vehicle_index
geometry_msgs/Vector3 alpha_v
